lib/useButter.js

```diff
@@ -123,18 +123,38 @@ export default function useButter(type, butterConfig) {
                     detail: errorDetail
```

Comment:
In some cases error detail might be not in `detail` prop. Should we cover somehow or current state is ok?
https://share.zight.com/8LuNRWmL

Reply:
thanks for this @marcinpece-apptension, I've updated the code to handle a more general set of error keys/values. Please give this all another check.